charts/stardog/templates/statefulset.yaml

```diff
@@ -73,9 +73,9 @@ spec:
           echo "Using existing zookeeper"
         {{ end }}
       containers:
-      - name: {{ .Release.Name }}-stardog
+      - name: {{ include "stardog.fullname" . }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
-        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        image: {{ .Values.image.registry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}
```

Comment:
The tests fail to start stardog with:

```
  Warning  InspectFailed     14s (x2 over 29s)      kubelet, default-a2ebce75-00cd-4a57-9d5a-b94bc233b63f  Failed to apply default image tag "https://registry.hub.docker.com/v1/repositories/stardog/stardog:latest": couldn't parse image reference "https://registry.hub.docker.com/v1/repositories/stardog/stardog:latest": invalid reference format
  Warning  Failed            14s (x2 over 29s)      kubelet, default-a2ebce75-00cd-4a57-9d5a-b94bc233b63f  Error: InvalidImageName
```

I think the registry is only needed in the image secret string and when those credentials are used the `{{ .Values.image.repository }}:{{ .Values.image.tag }}` will pull from there.

Reply:
Done. I've tested it manually pulling from dockerhub and the private jfrog repo and it works.
I wasn't able to add a test for the private repo as it requires the password in the config.